tests/integration/services/test_k8s.py

```diff
@@ -0,0 +1,401 @@
+import os
+from http import HTTPStatus
+
+import pytest
+from dotenv import load_dotenv
+from requests import Response
+
+from services.k8s import K8sClient
+
+# load env file if exists.
+load_dotenv(os.getenv("ENV_FILE", "../../../.env.evaluation"))
```

Comment:
can you load the `.env.test` instead of `env.evaluation`. 
You don't even load it is loaded by the settings.py script.

Reply:
I have updated it.